lib/OpenQA/Parser/Format/LTP.pm

```diff
@@ -30,7 +30,13 @@ sub parse {
         $t_name =~ s/:/_/g;
 
         $result->{result} = 'ok';
-        $result->{result} = 'fail' if $result->{test}->{result} !~ /pass/i || $result->{status} !~ /pass/i;
+
+        if ($result->{test}->{result} =~ /(fail|brok)/i || $result->{status} =~ /(fail|brok)/i) {
+            $result->{result} = 'fail';
+        }
+        elsif ($result->{test}->{result} =~ /conf/i || $result->{status} =~ /conf/i) {
+            $result->{result} = 'skip';
```

Comment:
What does conf mean?

We are not being charged per character :)

Reply:
`TCONF`. Remember we're parsing LTP API.

Reply:
> What does conf mean?

It meas the test started executing, but exited early because the system's configuration will not allow it to complete (e.g. the kernel feature being tested is missing).